MoreLinq/tt.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+[[ -e tt.sh ]] || { echo >&2 "Please cd into the script location before running it."; exit 1; }
```

Comment:
The reason I didn't go for `cd "$(dirname "$0")"` is to avoid the [so many ways you can skin this cat and still get it wrong by someone's book](https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within). I figured, leave the problem up to the caller when the invocation from the shell could be just `(cd MoreLinq; ./tt.sh)`. However, for all practical intent and purposes of this repo, I can imagine `cd "$(dirname "$0")"` is good enough and that we needn't worry ourselves about corner cases. But then this is something we should do for all `*.sh` shell scripts in the repo, don't you think?